src/main/java/org/pokecollect/dto/Pokecard.java

```diff
@@ -5,12 +5,19 @@
 import java.util.List;
 
 public @Data class Pokecard {
+    private final String id;
+    private final String name;
+    private final String level;
+    private final String hp;
+    private final List<String> types;
+    private final String description;
 
-    private String id;
-    private String name;
-    private String level;
-    private String hp;
-    private List<String> types;
-    private String description;
-
+    public Pokecard(String id, String name, String level, String hp, List<String> types, String description) {
```

Comment:
If you intend for Pokecard objects to be immutable (i.e., their state cannot be changed after creation), you can make the fields final and initialize them in a constructor. This can help ensure that the state of a Pokecard object remains consistent.